src/ethz_snow/config/snowConfig_default.yaml

```diff
@@ -1,36 +1,87 @@
+# temperature resolution within vial (homogeneous, spatial_1D, spatial_2D)
+temperature: spatial_2D
+
+# freezing configuration to be simulated (shelf, VISF, jacket)
+configuration: jacket
+
+# all vial-related parameters
 vial:
+  # define the geometry
   geometry:
-    # base shape of vial (currently only cube is accepted)
-    shape: cube
-    # length [m]
+    # base shape of vial (in the spatial version only cylinder is accepted)
+    shape: cylinder
```

Comment:
Won't this change to the default break snow and snowfall?

Reply:
It at least breaks the associated unit test 

```
FAILED tests/test_constants.py::test_warningWrongYamlSchema - assert 7.853981633974483e-07 == ((1 * 0.01) * 0.01)
```

This is due to the fact that the expectation was that the default shape is a cube

Reply:
We cannot really generalize the shape for all models, so my idea was that we keep cube as the default shape for snow and snowfall, but we change it to cylinder for snowing. Then the users have to change this depending on which model they want to use.